src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Overview/ThorchainSaversOverview.tsx

```diff
@@ -241,6 +242,29 @@ export const ThorchainSaversOverview: React.FC<OverviewProps> = ({
       .length,
   )
 
+  const isOpportunityMature = useMemo(() => {
+    if (!isRunePool) return true
+    if (!isRunePoolOpportunity(earnOpportunityData)) return false
+    if (!earnOpportunityData?.maturity) return false
+
+    const maturityDate = new Date(earnOpportunityData.maturity)
+    const currentDate = new Date()
+
+    return currentDate >= maturityDate
+  }, [earnOpportunityData, isRunePool])
+
+  const runepoolDaysLeftBeforeWithdrawal = useMemo(() => {
```

Comment:
Seconds over days, see comment in `disabledRunepoolWithdraw` copy

Reply:
Changed in [4abc315](https://github.com/shapeshift/web/pull/7472/commits/4abc3154365f8295c06c2fb383b10d762be64282)